packages/wallet/src/services/DelegationTracker/DelegationTracker.ts

```diff
@@ -68,6 +90,92 @@ export const certificateTransactionsWithEpochs = (
     )
   );
 
+export const createDelegationPortfolioTracker = ({
+  rewardAccounts$,
+  knownAddresses$,
+  utxoTracker
+}: DelegationPortfolioTrackerProps): Observable<Cip17Pool[]> => {
+  const balanceAllUtxosTracker = createBalanceByAddressTracker(utxoTracker);
+
+  const delegatedAccounts$ = rewardAccounts$.pipe(
+    map((rewardsAccounts) =>
+      rewardsAccounts.filter(
+        (account) => account.keyStatus === Cardano.StakeKeyStatus.Registered && account.delegatee?.nextNextEpoch
+      )
+    )
+  );
+
+  const hydratedDelegatedAccounts$ = delegatedAccounts$.pipe(
+    withLatestFrom(knownAddresses$),
+    map(([delegatedAccounts, knownAddresses]) =>
+      delegatedAccounts.map((delegatedAccount) => {
+        const groupedAddress = knownAddresses.find(
+          (knownAddr) => knownAddr.rewardAccount === delegatedAccount.address
+        )!;
+        return {
+          balance: createBalanceByAddressTracker(utxoTracker, groupedAddress.address),
```

Comment:
createBalanceByAddressTracker calculates only the utxo balance.
The rewards must be included in this calculation

Reply:
Done here: [da7fc70](https://github.com/input-output-hk/cardano-js-sdk/pull/735/commits/da7fc70df3a90dfce49211ecb21ae740b30fed39)